ffi/diplomat/Cargo.toml

```diff
@@ -31,31 +31,25 @@ all-features = true
 # Omit most optional dependency features from permutation testing
 skip_optional_dependencies = true
 # Bench feature gets tested separately and is only relevant for CI.
-# wearos/freertos/x86tiny are not relevant in normal environments,
+# x86tiny is not relevant in normal environments,
```

Comment:
Suggestion: get rid of the `x86tiny` feature in this crate and move it to the crate that needs it (staticlib?)